local/kcp/start.sh

```diff
@@ -20,7 +20,7 @@ set -o nounset
 # Parameters
 printf "The following optional parameters can be passed to the script:\n"
 printf "KCP_DIR: a directory with kcp source code, default to a git clone of kcp in the system temp directory\n"
-printf "KCP_BRANCH: the kcp branch to use. Mind that the script will do a git checkout, to a default release if the branch is not specified\n"
+printf "KCP_TAG: the kcp branch to use. Mind that the script will do a git checkout, to a default release if the branch is not specified\n"
```

Comment:
KCP_TAG: the kcp branch to use => KCP_TAG: the kcp tag to use
 if the branch is not specified =>  if the tag is not specified

Reply:
Fixed.